applications/GeoMechanicsApplication/custom_utilities/math_utilities.h

```diff
@@ -12,12 +12,12 @@
 
 #pragma once
 
-#include <vector>
-
 #include "includes/kratos_export_api.h"
 #include "includes/ublas_interface.h"
 
 #include <algorithm>
+#include <vector>
+#include <cstdlib>
```

Comment:
A non-blocking remark: since `std::abs` is no longer used in this header file, this include is--strictly speaking--not needed anymore. But I would only remove it if there are any other suggestions that you'd to incorporate (to avoid waiting for a long time for all pipelines to pass).